apps/kindergarten/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+
+from apps.kindergarten.views import KinderView
+
+urlpatterns = [
+    url(r'^(?P<pk>[0-9]+)$', KinderView.as_view()),
```

Comment:
add name for url 
like this https://docs.djangoproject.com/en/1.11/topics/http/urls/#examples